src/fsharp/PackageManager.Integration.fs

```diff
@@ -38,15 +38,17 @@ type PackageManager = {
     ToolName: string
     Name:string }
 
-let RegisteredPackageManagers : PackageManager list = 
+let RegisteredPackageManagers = lazy (
```

Comment:
> Then a single eval would be enough, right?

Like @smoothdeveloper said, it can certainly be decided later, but I was heading in the direction that this should be able to find new "package managers" in external libraries as they are referenced. So I wouldn't need to ship `amazingant.packageloader.dll` with fsi to use it, I could just `#r "some/path/to/amazingant.packageloader.dll"` and then use `#r "amazingant: ..."`. Not sure where/how type providers are detected now, but I'd imagine the code for them could be modified to also load these? The `RegisteredPackageManagers` list would probably need to be mutable rather than lazy to do that.